graphdatascience/graph_data_science.py

```diff
@@ -144,6 +149,12 @@ def __init__(
                 ):
                     warnings.warn(f"Could not initialize GDS Flight Server client: {e}")
 
+        if aura_db_connection_info and self._server_version >= ServerVersion(2, 4, 0):
+            aura_db_conneciton_info = AuraDbConnectionInfo(
+                aura_db_connection_info[0], aura_db_connection_info[1], self._config
```

Comment:
Will this always be the same config as for the other Neo4j driver? Or perhaps these are disjoint cases?

Reply:
You are right, they are disjoint. I used the default config now, just with a connection lifetime of 1 minute